### DISCUSSION TITLE
Using Silk.NET OpenGL API within Winforms?

### DISCUSSION THREAD

**Discussion by ghost** [dotnet/Silk.NET#1331] [General]
I have recently moved towards Silk.NET from OpenTK to stay updated.

Is there anything that is equivalent to the "GLControl" found in OpenTK so I can use Silk.NET OpenGL in a WinForms application? Any help on this subject would be greatly appreciated

Thank you!

**Comment by Beyley** [comment]
To my knowledge no one has created a winforms control for Silk.NET specifically, it may be possible to modify the OpenTK one to work with Silk.NET though

**Reply by ghost** [reply]
I'll have a dig around and see what I can come up with. Just wanted to know if there was already a solution before doing so... Thank you!

**Reply by Perksey** [reply]
We recently added WGL bindings with intentions of making a winforms control and WPF control, but none of the maintainers have time to do this right now.
